From c183b588eb285547ad9ac9c31429060a54bd9c68 Mon Sep 17 00:00:00 2001
From: Victoria Milhoan <vicki.milhoan@freescale.com>
Date: Wed, 10 Dec 2014 15:32:26 -0700
Subject: [PATCH] MLK-9980 Modify CAAM driver to hide algorithm registered
 notifications

The CAAM driver prints a message for each algorithm it registers
with the Crypto API. This patch hides the messages unless debug is
enabled.

Signed-off-by: Victoria Milhoan <vicki.milhoan@freescale.com>
(cherry picked from commit 84fcc913c4017d7c60ad19d07f277165b10e7848)
---
 drivers/crypto/caam/caamalg.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/crypto/caam/caamalg.c b/drivers/crypto/caam/caamalg.c
index 836e929..1785530 100644
--- a/drivers/crypto/caam/caamalg.c
+++ b/drivers/crypto/caam/caamalg.c
@@ -2384,8 +2384,10 @@ authencesn:
 			kfree(t_alg);
 		} else {
 			list_add_tail(&t_alg->entry, &alg_list);
+#ifdef DEBUG
 			dev_info(ctrldev, "%s\n",
 				 t_alg->crypto_alg.cra_driver_name);
+#endif
 
 			if (driver_algs[i].type == CRYPTO_ALG_TYPE_AEAD &&
 			    !memcmp(driver_algs[i].name, "authenc", 7) &&
-- 
1.8.0

